docs/pivoting.md

```diff
@@ -0,0 +1,143 @@
+# Move
```

Comment:
The file is called "pivoting". Can we make the file and title match?

Reply:
Done